### ISSUE TITLE
JAWS browse mode broken in dropdown dialogs

### ISSUE THREAD

**Issue by wkeese** [ibm-js/delite#511]
For a TooltipDialog opened from a HasDropDown (with ` dropDownType="dialog"`), the JAWS Ctrl-Up and Ctrl-Down key doesn't work, so a JAWS user can't read the TooltipDialog's boilerplate text.
